backend/src/User.ts

```diff
@@ -16,22 +13,12 @@ export class User {
     signedIn: 'signed-in',
   }
 
-  username: string
-  authHash: string
+  username: string = ''
+  authHash: string = ''
   signedIn: boolean = false
-  // configFile: JSONFile<ConfigFile>
-  private userFile: JSONFile<UserCredentials>
-
+ 
   constructor() {
-    this.userFile = new JSONFile<UserCredentials>(path.join(environment.userPath, 'user.json'))
-    // this.configFile = new JSONFile<UserCredentials>(path.join(environment.userPath, 'user.json'))
-    const user = this.userFile.read()
-
-    d('Reading user credentials:', { user })
-    Logger.info('Setting user:', { username: user && user.username })
 
-    this.username = user?.username || ''
-    this.authHash = user?.authHash || ''
     if (this.hasCredentials) this.signedIn = true
```

Comment:
Since this starts up right away I don't think this will ever get called, so it should be safe to remove as well. The whole constructor call can go.